node/service/src/chain_spec/frequency_paseo.rs

```diff
@@ -67,108 +54,6 @@ pub fn local_paseo_testnet_config() -> ChainSpec {
 	.with_protocol_id("frequency-paseo-local")
 	.with_properties(properties)
 	.with_chain_type(ChainType::Local)
-	.with_genesis_config(testnet_genesis(
-		// initial collators.
-		vec![
-			(
-				get_account_id_from_seed::<sr25519::Public>("Alice"),
-				get_collator_keys_from_seed("Alice"),
-			),
-			(
-				get_account_id_from_seed::<sr25519::Public>("Bob"),
-				get_collator_keys_from_seed("Bob"),
-			),
-		],
-		// Sudo
-		Some(get_account_id_from_seed::<sr25519::Public>("Alice")),
-		// Endowed Accounts
-		vec![
-			get_account_id_from_seed::<sr25519::Public>("Alice"),
-			get_account_id_from_seed::<sr25519::Public>("Bob"),
-			get_account_id_from_seed::<sr25519::Public>("Charlie"),
-			get_account_id_from_seed::<sr25519::Public>("Dave"),
-			get_account_id_from_seed::<sr25519::Public>("Eve"),
-			get_account_id_from_seed::<sr25519::Public>("Ferdie"),
-			get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
-			common_runtime::constants::TREASURY_PALLET_ID.into_account_truncating(),
-		],
-		// Council members
-		vec![
-			get_account_id_from_seed::<sr25519::Public>("Alice"),
-			get_account_id_from_seed::<sr25519::Public>("Charlie"),
-			get_account_id_from_seed::<sr25519::Public>("Eve"),
-		],
-		// Technical Committee members
-		vec![
-			get_account_id_from_seed::<sr25519::Public>("Bob"),
-			get_account_id_from_seed::<sr25519::Public>("Dave"),
-		],
-		// ParaId
-		2000.into(),
-	))
+	.with_genesis_config_preset_name("frequency-local")
```

Comment:
question: the filename is `frequency_paseo.rs` but the config name in here is `frequency-local `. Just wanted to make sure we are doing this deliberately.

Reply:
Thank you for pointing that out! I see what you mean, and you’re right — the naming might need a bit of updating in a future PR. For now, though, this file is specifically for Paseo local, which is referred to as frequency-local.